src-test/org/etools/j1939_84/bus/j1939/packets/DTCLampStatusTest.java

```diff
@@ -121,6 +121,7 @@ public void testEquals() {
                 0x55), null);
         assertEquals(true, instance.equals(instance2));
         assertEquals(false, instance.equals(instance3));
+        // FIXME what is this supposed to be doing?
```

Comment:
This is checking that equals is false when compared to an object of a different class.